backend/cmd/plugins.go

```diff
@@ -29,20 +29,29 @@ func defaultPluginDir() string {
 	//  - "./.plugins" if it exists.
 	//  - ~/.config/Headlamp/plugins exists or it can be made
 	//  - "./.plugins" if the ~/.config/Headlamp/plugins can't be made.
+	// Windows: %APPDATA%\Headlamp\Config\plugins
+	//   (for example, C:\Users\USERNAME\AppData\Roaming\Headlamp\Config\plugins)
 	pluginDirDefault := "./.plugins"
 
 	if folderExists(pluginDirDefault) {
 		return pluginDirDefault
 	}
 
+	// https://www.npmjs.com/package/env-paths
+	// https://pkg.go.dev/os#UserConfigDir
 	userConfigDir, err := os.UserConfigDir()
 	if err != nil {
 		log.Printf("error getting user config dir: %s\n", err)
 
 		return pluginDirDefault
 	}
 
-	pluginsConfigDir := path.Join(userConfigDir, "Headlamp", "plugins")
+	pluginsConfigDir := filepath.Join(userConfigDir, "Headlamp", "plugins")
```

Comment:
Nit: I usually try to avoid extra memory usage (unless golang is smarter than I think).
So I'd not join the path here until I know we're not on Windows.